### PR TITLE
Explorer

### PR DESCRIPTION
**Pull request by ManuelBilbao**
Add an explorer for the Blockchain

### CODE REVIEW

File: explorer/.github/FUNDING.yml
Comment (fborello-lambda) [inline explorer/.github/FUNDING.yml:1]: can this file be deleted?

File: explorer/.github/workflows/config-check.yaml
Comment (fborello-lambda) [inline explorer/.github/workflows/config-check.yaml:1]: can this file be deleted?

File: explorer/.github/workflows/docker.yaml
Comment (fborello-lambda) [inline explorer/.github/workflows/docker.yaml:1]: can this file be deleted?

File: explorer/.github/workflows/mainnet-deploy.yaml.disabled
Comment (fborello-lambda) [inline explorer/.github/workflows/mainnet-deploy.yaml.disabled:1]: can this file be deleted?

**Comment by fborello-lambda** [comment]
Can we delete the `.github` folder?

**Comment by ManuelBilbao** [comment]
Done

**Comment by JulianGCalderon** [comment]
Which files need review / were changed by us?

File: explorer/Makefile
Comment (JulianGCalderon) [inline explorer/Makefile:1]: Maybe we can have a make prod to run in the production testnet.

**Comment by ManuelBilbao** [comment]
I'd test if the explorer runs correctly more than a code review. As far as I remember, the only files changed/added are `Makefile` and `chains/mainnet/alignedlayer.json`

**Review by JulianGCalderon — COMMENTED** [review]
Should we remove the cosmos mainnet explorer?

**Comment by JulianGCalderon** [comment]
> I'd test if the explorer runs correctly more than a code review. As far as I remember, the only files changed/added are `Makefile` and `chains/mainnet/alignedlayer.json`

Thanks. Turns out I can select a range of commits to review, avoiding the first commit (cloned code)

**Comment by ManuelBilbao** [comment]
> Should we remove the cosmos mainnet explorer?

@MauroToscano said we should keep it

File: explorer/Makefile
Comment (ManuelBilbao) [reply explorer/Makefile:1]: We have the `make build` target, which compiles the app and let it ready to serve on prod. As it's a static web app, there's no more to do, I think

**Review by JulianGCalderon — APPROVED** [review]
LGTM

**Review by JulianGCalderon — APPROVED** [review]
LGTM

**Review by fborello-lambda — APPROVED** [review]

**Review by JulianGCalderon — APPROVED** [review]
LGTM

**Pull request merged**